pkg/alerts/alertsqlite/alerts_sqlite.go

```diff
@@ -103,6 +116,23 @@ func isValid(str string) bool {
 	return str != "" && str != "*"
 }
 
+func retry(operation func(attemptCount int) error) error {
+	var err error
+	for i := 0; i < maxRetries; i++ {
+		err = operation(i + 1)
+		if err == nil {
+			return nil
+		}
+		if strings.Contains(err.Error(), "database is locked") {
+			log.Warnf("retry: database is locked, retrying attempt %d. Error=%v", i+1, err)
+			time.Sleep(time.Duration(math.Pow(2, float64(i))) * time.Millisecond * 50)
```

Comment:
nit: we might want `50 ms` to be a constant defined next to `maxRetries` at the top of the file. What do you think?